vite.config.ts

```diff
@@ -0,0 +1,9 @@
+import { sveltekit } from '@sveltejs/kit/vite';
```

Comment:
I'd probably leave this as `vite.config.js`. Bundling the config file introduces a lot of extra complexity and we've tried to avoid it in most of the other `sveltejs` projects

Reply:
Sounds good 👍